internal/resources/kubernetessecret/resource_secret.go

```diff
@@ -0,0 +1,111 @@
+/*
+Copyright © 2023 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package kubernetessecret
+
+import (
+	"github.com/go-openapi/strfmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	secretmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/kubernetessecret/cluster"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/kubernetessecret/scope"
+)
+
+func ResourceSecret() *schema.Resource {
+	return &schema.Resource{
+		Schema: secretSchema,
+	}
+}
+
+var secretSchema = map[string]*schema.Schema{
+	NameKey: {
+		Type:     schema.TypeString,
+		Required: true,
+		ForceNew: true,
+	},
+	NamespaceNameKey: {
+		Type:     schema.TypeString,
+		Required: true,
+		ForceNew: true,
+	},
+	OrgIdKey: {
```

Comment:
it's defined in the schema, not sure

@ishangupta-ds is it can be removed. ?

Reply:
@Vasundharashukla, we need the `org_id` outside of the scope because as per the current state and in near future we will be supporting only cluster and cluster group scopes. 

Since this organization ID would have been a common field across both these planned scopes, I think it is better to keep it outside the scope block.

WDYT?